src/animationInputMappers.ts

```diff
@@ -1,79 +1,32 @@
-import { AnimationInput, AnimationOptions, normalizedAnimation, NormalizedAnimation } from './AnimationInput.js';
-import { ToggleTransitions } from './hooks/useTransitioningToggle.js';
+import { AnimationInput, normalizedAnimation } from './AnimationInput';
+import { ToggleTransitions } from './hooks/useTransitioningToggle';
 
 /**
- * Returns an edited animation that runs it's last keyframe infinitely (until interrupted)
- * Useful for controlled animations that are tied to state rather than transitions
+ * Helper to create a complete toggling animation by reversing a given togglingOn animation
  *
  * The edits are:
- * - set iterations: 'Infinity' on the last Keyframe in the Keyframes list
  * - set fill: 'forwards' on the Animation options
+ * - togglingOff is exactly the togglingOn animations with the keyframes array reversed. If an offset values is set at keyframes[i], it is swapped with the offset from keyframes[keyframes.length - (i + 1)]
  */
-export const toPersistedAnimation = (toPersist: AnimationInput): NormalizedAnimation => {
```

Comment:
Got rid of these mappers except for toToggleAnimations. With animationOptions prop, no need to set up common styles for each animation, which is all the rest were doing